### DISCUSSION TITLE
Multi Prompts ???

### DISCUSSION THREAD

**Discussion by PeterJared** [lllyasviel/Fooocus#1659] [General]
This line in the installation page:

"Multi Prompts (::) | Just use multiple lines of prompts"

Suggests to me that I can create a sequence of prompts in one single prompt text box and as long the number of images requested allows it, it will generate automatically each image with a different prompt. But I tried this and only one prompt is used:

Prompt 1::
A beautiful sunset over the ocean.

Prompt 2::
A majestic mountain range covered in snow.

Prompt 3::
A serene forest with a river running through it.

Prompt 4::
A bustling city street at night.

Prompt 5::
A peaceful countryside with a farm and animals.

I tried other combinations with no results.

**Comment by xhoxye** [comment]
I also wonder how it should be used.

**Comment by PeterJared** [comment]
I noticed the wildcards can do something quite similar to what I explain above. The multi prompts in that command above is in reality something completely different, as it makes juxtaposition instead of mixing as normal of terms.

**Comment by CraigOndrak** [comment]
I'm also trying to figure out a way to feed multiple prompts so my computer keeps generating while I sleep. Can someone experienced please provide some insights?

**Reply by mashb1t** [reply]
https://github.com/lllyasviel/Fooocus/pull/1503 can be used together with right-clicking generate > "generate forever" to continuously generate images with multiple promps.

Some people are also currently working on a queue feature, see https://github.com/lllyasviel/Fooocus/issues/1664.

**Reply by CraigOndrak** [reply]
Great Stuff.. thank you for "Generate Forever" too. Didn't know about its existence .. will give it a try.

**Reply by CraigOndrak** [reply]
Hey @mashb1t ,

I've been trying the arrays in main prompt as per #1503 but not getting success. I'm tried a simple prompt, a [[man, woman, cat]] sitting [[in a park, on the beach, in a cafe]] and got some weird results. Am I missing something mate? 

![image](https://github.com/lllyasviel/Fooocus/assets/154491748/2601a02c-8346-43e5-9654-5922f31f43f3)
![image](https://github.com/lllyasviel/Fooocus/assets/154491748/3054bf4d-5145-4875-8c1c-f9f6ea1faa9e)

**Reply by mashb1t** [reply]
This feature has not yet been reviewed/merged. Please get in touch with the creator of the PR @flanneryhbh or add a comment in the PR.
